pkgs/by-name/po/popping-dikr/package.nix

```diff
@@ -0,0 +1,38 @@
+# بسم الله الرحمن الرحيم
+# la ilaha illa Allah Mohammed Rassoul Allah
```

Comment:
I dont kown what is the means of it, but just dont add comment not in english and meanless.

Reply:
can I please leave the comments? please?